pages/Workflow.md

```diff
@@ -10,14 +10,48 @@ A Developer wishing to design and deploy his/her own data services with Hydra, [
+Controller: It’s a central hydra server which has the powers of starting, stopping drones and also giving instructions to them.
+Flock of drones: A group of drones which aim to detect fire or abnormal heat spots in a given geographical area, each drone can be active or not active at a given moment. They receive instructions from Controller and update controller every 15 sec.
+
+Assuming that you have already visited the demo. You might have come across various sections and should be curious what each one of them really means. So let’s study them one by one. We’ll go from left to right, would first study all the sections at left side to the map and then would study sections right to it. So here we go!
+
+First up there is input field which you can use to give instructions to individual drone, for eg you can change the speed of the drone by writing Set Drone 1 speed 10. As you might have already guessed. This message is first sent to controller via GUI and then controller takes some action depending on input supplied to it. 
+
+Next up is the Refresh Button, which you can use to reload active drones at any moment.
+
+Next up is the HTTP-API Logs Section. When some instructions are sent to drone they are sent via HTTP requests which can be either PUT or POST. This section logs all that activity. Now we are done with our left side and lets hop over to the right side and see what we can find over there.
+
+In this side, we've got two log sections similar to controller log we talked about moments ago. Remember we logged type of request and target over there, and in the first section we log purpose of that request. Now you may be wondering what do mean by purpose where does it come from? So For your information it’s the same thing which you typed in the input field we discussed earlier and sent to controller. As we already know that each drone updates controller every 15 sec. So in the second/last section we log updates sent from drone to controller.
+
+Now you might I have at least got an idea what every section means in the demo and basically what the demo is all about. But you must be having one question in your mind at the moment viz. Where is hydra? So the answer is it’s everywhere.It’s just that We haven’t discussed about it yet. Let’s begin!
+
+The Web API discussed above has to expose representations of drones, datastreams, commands (sent via GUI), states (whether active or inactive), messages, log entries and areas. In order to make  interaction of these resources (data), client needs to be aware of operations supported by the server. In normal websites (websites not using hydra) such resources are made available via links/forms. But the problem is that they are not machine readable. Data represented in natural language can easily understood by humans but for machines/computers it’s just a collection of random bits which doesn’t convey any meaning to them. That’s where hydra comes to the rescue and we use it to resolve this issue.
```

Comment:
` Data represented in natural language can easily understood by humans but for machines/computers it’s just a collection of random bits which doesn’t convey any meaning to them`
Natural language is still random bits to computers.